### PR TITLE
Trades graph

### PR DESCRIPTION
**Pull request by Soimil**

### CODE REVIEW

File: apps/hefty/lib/hefty/trades.ex [hunk 1]
```diff
@@ -234,4 +234,50 @@ defmodule Hefty.Trades do
...
+
+  def get_trades_dates() do
+    query =
+      "SELECT to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') AS date " <>
+        "FROM trades GROUP BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') " <>
+        "ORDER BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') DESC;"
+
+    result =
+      Ecto.Adapters.SQL.query!(Hefty.Repo, query).rows
+      |> Enum.map(fn elm ->
```
Comment (Cinderella-Man) [inline apps/hefty/lib/hefty/trades.ex]: Could this be `|> Enum.map(&List.fist/1)` ?

File: apps/hefty/lib/hefty/trades.ex [hunk 2]
```diff
@@ -234,4 +234,50 @@ defmodule Hefty.Trades do
...
+        "GROUP BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD'), symbol " <>
+        "ORDER BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') DESC, symbol ASC;"
+
+    result =
+      Ecto.Adapters.SQL.query!(Hefty.Repo, query).rows
+      |> Enum.map(fn [symbol, total, date] -> ["#{symbol}": ["#{date}": total]] end)
+
+    days = Hefty.Utils.Datetime.get_last_days(30)
+
+    return_values =
```
Comment (Cinderella-Man) [inline apps/hefty/lib/hefty/trades.ex]: `return_values` is not required here - it can be just last command

**Review by Cinderella-Man — CHANGES_REQUESTED** [review]
Looks good beside couple tiny improvements

File: apps/hefty/lib/hefty/trades.ex (hunk 2 above)
Comment (Soimil) [reply apps/hefty/lib/hefty/trades.ex]: True, and done.

File: apps/hefty/lib/hefty/trades.ex (hunk 1 above)
Comment (Soimil) [reply apps/hefty/lib/hefty/trades.ex]: Updated. Thx

**Pull request merged**